core-common/src/main/java/org/mapstruct/MapperConfig.java

```diff
@@ -135,4 +135,12 @@
      */
     MappingInheritanceStrategy mappingInheritanceStrategy()
         default MappingInheritanceStrategy.EXPLICIT;
+
+    /**
+     * Decide whether we should check null or hasX method before mapping.
+     * It is global, can only override the one on {@link Mapping} if it is UNDEFINED.
```

Comment:
I am a bit confused by this message; How is the order of precedence? I'd expect the value on `@Mapping` take precedence over `@Mapper` over `@MapperConfig`. Is that what happens?